buffalo/cmd/build.go

```diff
@@ -40,6 +40,14 @@ var xbuildCmd = &cobra.Command{
 			b.Tags = append(b.Tags, buildOptions.Tags)
 		}
 
+		if b.WithYarn {
+			found, err := YarnScriptExists("buffalo:dev")
+			if err != nil {
+				logrus.Fatal(err)
+			}
+			b.WithYarn = found
```

Comment:
This feels like I'm abusing a flag that just happens to be in the right place. This 'makes it work' but should I do something else to pass this information in to the builder?